source/Locales/DistributiveLattice/Ideal.lagda

```diff
@@ -184,6 +187,8 @@ module PrincipalIdeals (L : DistributiveLattice 𝓤) where
 
 \end{code}
 
+Some nice syntax.
+
 \begin{code}
 
  syntax principal-ideal x = ↓ x
```

Comment:
I don't think `syntax` is needed for this.

Reply:
Tried turning this into a definition with the same precedence, but it did not work. I will fix it in the future.